website/docs/r/app_service_environment.html.markdown

```diff
@@ -58,7 +59,9 @@ resource "azurerm_app_service_environment" "example" {
 
 ~> **NOTE** a /24 or larger CIDR is required. Once associated with an ASE this size cannot be changed.
 
-* `internal_load_balancing_mode` - (Optional) Specifies which endpoints to serve internally in the Virtual Network for the App Service Environment. Possible values are `None`, `Web` and `Publishing`. Defaults to `None`.
+* `internal_load_balancing_mode` - (Optional) Specifies which endpoints to serve internally in the Virtual Network for the App Service Environment. Possible values are `None`, `Web`, `Publishing` and combined value `"Web, Publishing"`. Defaults to `None`.
+
+~> **NOTE** You should prefer to select value either `None` or combined value `"Web, Publishing"` as the other values might become deprecated in the future.
```

Comment:
Is this documented / published somewhere? If not official, it's probably as well to leave this out for the time being.

Reply:
It is info from Microsoft ASE Program Manager - Christina - (my internal communication). No any official statement I'm aware of. Let me know what you prefer I'll change it accordingly.